spec/lib/sequent/core/command_service_spec.rb

```diff
@@ -96,8 +96,8 @@ def password
       )
     end
 
-    context 'and dutch as error locale' do
-      before { Sequent.configuration.error_locale_resolver = -> { :nl } }
+    context 'and dutch as the locale' do
+      before { I18n.locale = :nl }
```

Comment:
I think this is also to test the `Sequent.configuration.error_locale_resolver` ?

Reply:
Why is this different from I18n.locale?

Reply:
This is for translating **just the command validation errors** in the correct local. So other things will still use the default translation. We added this to return errors in english in an api for instance, but still be able to use another language as language for the normal flow.

Reply:
Why did it fail?

Reply:
nvm, fixed it :)

Reply:
Apparently I18n.with_locale now doesn't cache the validation messages, so we had to move the block to where it's called.